lib/ibm_speech_to_text.ex

```diff
@@ -65,32 +64,32 @@ defmodule Membrane.Element.IBMSpeechToText do
         Keyword.merge(rec_opts, content_type: :flac, inactivity_timeout: -1)
       end)
 
-    {:ok, state}
+    {[], state}
   end
 
   @impl true
-  def handle_prepared_to_playing(_ctx, state) do
+  def handle_playing(_ctx, state) do
     with {:ok, pid} <- Client.start_link(state.region, state.api_key, state.client_options) do
```

Comment:
This has to be started with https://hexdocs.pm/membrane_core/Membrane.UtilitySupervisor.html